### DISCUSSION TITLE
Duplicate id insertion [Oct 12]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6744] [Discord Threads]
> 👨‍🦲 **dstala**

**Comment by o1lab** [comment]
> 👨‍💻 **pranavxc**  
#### Option 2 will be the faster way to do it, if insert failed and error is duplicate id error  
#  
> 👨‍💻 **pranavxc**  
#### With option 1, you have to use the exist or read api first and then insert can be done]  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1150998091714015323)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
